### DISCUSSION TITLE
Unable to launch Exult Studio

### DISCUSSION THREAD

**Discussion by Crowley9** [exult/exult#286] [General]
I got back to Exult after a long while. I install the latest 1.9 snapshots of both Exult and Exult Studio, start the game, press control-alt-m, and... nothing happens. Looking through the documentation, everything should be in order. I have Studio installed in the same directory as Exult, I am running the game in windowed mode, cheats are enabled. I have absolutely no idea what is the problem here.

**Comment by DominusExult** [comment]
Nothing happens as in not even the red grid will show?
What is your OS?

**Comment by Crowley9** [comment]
Windows 10, and nothing will happen. No red grid, no Exult Studio window being opened.

**Comment by DominusExult** [comment]
You have to have cheats enabled for this :) (we might have to rethink this approach, though)

**Comment by Crowley9** [comment]
I do have cheats enabled, as I already mentioned in the first post.

**Comment by DominusExult** [comment]
Sorry, I read it but was too focused on other things.
It's really odd as the only time Exult does nothing is when cheats are not enabled.
Even if Studio is not in the same folder, Exult should go into map edit mode (red grid).
Can you check the exult.cfg if cheats are enabled in that, too?

And then whether Exult 1.8 works 🤷‍♂️

**Comment by Crowley9** [comment]
1.8 doesn't work either. Cheats are enabled in exult.cfg, and other cheat commands work just fine. I tried checking the logs for any error messages. No idea if these have anything to do with what is going on:

from stdout:

Looking for 'blackgate' at 'D:\Ultimat\ULTIMA7'... found game with identity 'FORGE'
Looking for 'forgeofvirtue' at './forgeofvirtue'... but it wasn't there.
Looking for 'serpentisle' at 'D:\Ultimat\Serpent'... found game with identity 'SILVER SEED'
Looking for 'silverseed' at './silverseed'... but it wasn't there.
Looking for 'serpentbeta' at './serpentbeta'... but it wasn't there.
Black Gate   : not found (<BLACKGATE_STATIC>\.)
Forge of Virtue   : found
exult_bg.flx : found
Serpent Isle   : not found (<SERPENTISLE_STATIC>\.)
Silver Seed   : found
exult_si.flx : found

from stderr:

Could not open file '.\forgeofvirtue\static\mainshp.flx
Could not open file '.\silverseed\static\mainshp.flx
Could not open file '.\serpentbeta\static\mainshp.flx

Tried running Exult Studio on its own, just in case. Didn't get anywhere, but here's some stuff from its logs:

from studio_out:

Looking for CSS at 'data/exult_studio.css'... but it wasn't there.
Looking for 'blackgate' at 'D:\Ultimat\ULTIMA7'... found game with identity 'FORGE'
Looking for 'forgeofvirtue' at './forgeofvirtue'... but it wasn't there.
Looking for 'serpentisle' at 'D:\Ultimat\Serpent'... found game with identity 'SILVER SEED'
Looking for 'silverseed' at './silverseed'... but it wasn't there.
Looking for 'serpentbeta' at './serpentbeta'... but it wasn't there.
Black Gate   : not found (<BLACKGATE_STATIC>\.)
Forge of Virtue   : found
exult_bg.flx : found
Serpent Isle   : not found (<SERPENTISLE_STATIC>\.)
Silver Seed   : found
exult_si.flx : found

from studio_err:

Could not open file '.\forgeofvirtue\static\mainshp.flx
Could not open file '.\silverseed\static\mainshp.flx
Could not open file '.\serpentbeta\static\mainshp.flx
Warning, trailing slash in path: "<BLACKGATE_STATIC>\"
Warning, trailing slash in path: "<SERPENTISLE_STATIC>\"
Error opening temporary file in gamedat dir with port number.

Also a bunch of "error sending to server" messages there.

**Reply by DominusExult** [reply]
Now the only thing I can think of is that some other program hijacks the shortcut ctrl-alt-m.
Apparently Nvidia's shadowplay hijacks it (but it could be anything else): https://superuser.com/questions/1282799/what-is-preventing-ctrl-alt-m-from-reaching-intellij-on-windows-10

**Reply by Crowley9** [reply]
> Now the only thing I can think of is that some other program hijacks the shortcut ctrl-alt-m. Apparently Nvidia's shadowplay hijacks it (but it could be anything else): https://superuser.com/questions/1282799/what-is-preventing-ctrl-alt-m-from-reaching-intellij-on-windows-10

Ooh, yes, that did it! Thank you!

**Reply by DominusExult** [reply]
Phew :)
I should have remembered, there was someone else with that problem a while ago on the old forum.

Nice to see you back! Sorry for having to kill the old forum
